metagov/metagov/settings.py

```diff
@@ -79,6 +80,28 @@
     "HIDE_HOSTNAME": False,
 }
 
+default_val = "testing-value" if TESTING else None
+
+METAGOV_SETTINGS = {
```

Comment:
PolicyKit settings.py looks like https://github.com/mashton/policykit/blob/bf600d205cdfe9a1aa58ede501bfb54f0ffc84b9/policykit/policykit/settings.py#L85-L97

Reply:
This looks a LOT better than what we're currently doing!